packages/seo-store/src/form/slice/social/twitter.js

```diff
@@ -0,0 +1,46 @@
+const { createSlice } = require( "@reduxjs/toolkit" );
+import { get } from "lodash";
+
+/**
+ * Sets the default state.
+ *
+ * @param {Object} props The properties to use.
+ */
+export const defaultTwitterState = {
+	form: {
+		social: {
+			twitter: {
+				title: "",
+				description: "",
+				image: {},
+			},
+		},
+	},
+};
```

Comment:
The default state should look like this:
```
export const defaultTwitterState = {
	title: "",
	description: "",
	image: {},
};
```

Otherwise, we will have an incorrect structure in the store: see screenshot
<img width="484" alt="Screenshot 2022-04-11 at 16 22 26" src="https://user-images.githubusercontent.com/48715883/162761818-5b8d3d74-53a5-458f-b02e-29003b5070e3.png">